core/templates/dev/head/dashboard/notifications_dashboard.html

```diff
@@ -26,46 +26,53 @@ <h2 class="oppia-page-heading">Notifications</h2>
+              <div class="oppia-notification-last-updated"><[getLocaleAbbreviatedDatetimeString(notification.last_updated_ms)]></div>
+              <div class="clearfix"></div>
+            </div>
 
-      <div ng-if="recentNotifications.length > 0">
-        <table class="table">
-          <tr>
-            <th></th>
-            <th></th>
-            <th>Exploration</th>
-            <th>Last updated</th>
-            <th>by</th>
-            <th>Subject</th>
-          </tr>
-          <tr ng-repeat="notification in recentNotifications"
-              ng-class="{'oppia-dashboard-row-recent': lastSeenMsec && lastSeenMsec < notification.last_updated_ms && notification.author_username !== currentUsername, 'oppia-dashboard-row': !lastSeenMsec || lastSeenMsec >= notification.last_updated_ms || notification.author_username === currentUsername}"
-              ng-click="navigateToItem(notification.activity_id, notification.type)">
-            <td>
-              <span class="label label-primary"
-                    ng-if="lastSeenMsec && lastSeenMsec < notification.last_updated_ms && notification.author_username !== currentUsername">
-                New
-              </span>
-            </td>
-            <td>
-              <span ng-if="notification.type === 'exploration_commit'"
-                    class="glyphicon glyphicon-flag"
-                    title="Exploration commit"></span>
-              <span ng-if="notification.type === 'feedback_thread'"
-                    class="glyphicon glyphicon-comment"
-                    title="Feedback thread"></span>
-            </td>
-            <td><[notification.activity_title]></td>
-            <td><[getLocaleAbbreviatedDatetimeString(notification.last_updated_ms)]></td>
-            <td><[notification.author_username]></td>
-            <td><[notification.subject]></td>
-          </tr>
-        </table>
+            <div ng-if="notification.type === 'exploration_commit'" ng-init="changeExplorationCommitNotificationColor();" class="oppia-exploration-commit-notification">
+              <div>
+                <span class="label label-primary"
+                      ng-if="lastSeenMsec && lastSeenMsec < notification.last_updated_ms && notification.author_username !== currentUsername">
+                  New
+                </span>
+              </div>
+              <div class="oppia-notification-type">
+                <span class="glyphicon glyphicon-pencil"
+                      title="Exploration commit"></span>
+              </div>
+                <div ng-if="notification.type === 'exploration_commit'">
+                  <div class="oppia-notification-author-username"><strong><a ng-click="navigateToProfile($event, notification.author_username)"><[notification.author_username]></a></strong><span ng-if="!notification.author_username">Anonymous</span> made a new commit to<span class="oppia-notification-activity-title"><[notification.activity_title]></span></div>
```

Comment:
Drop /div to next line.

But really, I don't like all this duplication. It's only the icon, the link, and a bit of the message that's changing, right? Better to have a single branch, and condition only for those three elements, rather than writing huge 'if' clauses (similar to the sort of thing that was done previously).